soepy/python/soepy_config.py

```diff
@@ -9,6 +10,6 @@
 np.testing.assert_equal(sys.version_info[1] >= 5, True)
 
 # We rely on relative paths throughout the package.
-ROOT_DIR = os.path.dirname(os.path.realpath(__file__))
-PACKAGE_DIR = ROOT_DIR.replace("/python", "")
-TEST_RESOURCES_DIR = PACKAGE_DIR + "/test/resources"
+ROOT_DIR = Path.cwd()
```

Comment:
This line changes the ROOT_DIR variable to a relative path. Therefore the actual path which is imported into the test files is dependent on the directory from which the test battery is executed. This induces that our test battery fails as soon as the imported directory no longer corresponds to "real" one. @peisenha : Since i haven't worked with pathlib so far i am definitely not the most qualified person to provide an advice regarding this topic. Would you mind to provide some help ?

Reply:
It appears that @tobiasraabe has a nice setup https://github.com/OpenSourceEconomics/respy/blob/extract-covariates/respy/python/shared/shared_constants.py. If there is further need to discuss, please put on our agenda for our Wednesday meetup.

Reply:
@peisenha , I was aware of the link above, but I am lost on how to adapt the very first line to my setup: "ROOT_DIR = Path(respy.__path__[0])" to my setup. I am under the impression that with Tobias' code it works after "import respy", but I cannot do "import soepy"..

@SeBecker , thank you, at least I now know what the problem is.